docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -27,6 +27,46 @@ kubectl label namespace default knative-eventing-injection=enabled
    with the appropriate permissions, you need to modify the
```

Comment:
We should add a link to the files you pulled the .yaml from up at the top. At the end of line 2, you could add,
```
The code for the following files can be found in the [/kubernetes-event-source/](https://github.com/knative/docs/tree/master/docs/eventing/samples/kubernetes-event-source) directory.
```